src/wallet/rpcdump.cpp

```diff
@@ -1768,20 +1758,20 @@ RPCHelpMan listdescriptors()
         },
         [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
 {
-    std::shared_ptr<CWallet> const wallet = GetWalletForJSONRPCRequest(request);
-    if (!wallet) return NullUniValue;
+    std::shared_ptr<CWallet> const pwallet = GetWalletForJSONRPCRequest(request);
```

Comment:
Our style is no longer to use `pthing` as the name for pointer variables, but I guess you did this to be consistent with the other functions in this file.

Reply:
Yea it's done to keep the diff to a minimum, otherwise we'd also have to rename all the pwallet usage in each function.